File: MORM.Java/src/MORM/Java/Models/Venda/Ncmsubst.java
package MORM.Java.Models.Venda;

import java.util.Date;
import MORM.Java.Classes.*;
import MORM.Java.Classes.Mapping.*;

@TabelaAttribute(Nome = "NCMSUBST")
public class Ncmsubst extends CollectionItem
{
    @CampoAttribute(Campo = "UF_ORIGEM", Tipo = CampoTipo.Key)
    public String Uf_Origem;
    @CampoAttribute(Campo = "UF_DESTINO", Tipo = CampoTipo.Key)
    public String Uf_Destino;
    @CampoAttribute(Campo = "CD_NCM", Tipo = CampoTipo.Key)
    public String Cd_Ncm;
    @CampoAttribute(Campo = "U_VERSION", Tipo = CampoTipo.Nul)
    public String U_Version;
    @CampoAttribute(Campo = "CD_OPERADOR", Tipo = CampoTipo.Nul)
    public Integer Cd_Operador;
    @CampoAttribute(Campo = "DT_CADASTRO", Tipo = CampoTipo.Nul)
    public Date Dt_Cadastro;
    @CampoAttribute(Campo = "CD_CEST", Tipo = CampoTipo.Nul)
    public String Cd_Cest;
}

File: MORM.CSharp/MORM.Dominio/Models/Venda/Ncmsubst.cs
using MORM.Util.Atributos;
using MORM.Util.Classes;
using System;

namespace MORM.CSharp.Models
{
    [Tabela("NCMSUBST")]
    public class Ncmsubst : CollectionItem
    {
        [Campo("UF_ORIGEM", CampoTipo.Key)]
        public string Uf_Origem { get; set; }
        [Campo("UF_DESTINO", CampoTipo.Key)]
        public string Uf_Destino { get; set; }
        [Campo("CD_NCM", CampoTipo.Key)]
        public string Cd_Ncm { get; set; }
        [Campo("U_VERSION", CampoTipo.Nul)]
        public string U_Version { get; set; }
        [Campo("CD_OPERADOR", CampoTipo.Nul)]
        public int Cd_Operador { get; set; }
        [Campo("DT_CADASTRO", CampoTipo.Nul)]
        public DateTime Dt_Cadastro { get; set; }
        [Campo("CD_CEST", CampoTipo.Nul)]
        public string Cd_Cest { get; set; }
    }
}